.github/workflows/ci.yml

```diff
@@ -0,0 +1,93 @@
+# Copyright 2025 Accenture
+
+name: CI
+
+on:
+  push:
+    branches: ["main"]
+  pull_request:
+    branches: ["*"]
+  workflow_dispatch:
+
+jobs:
+  bazel-build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+
+      - name: Install Protoc
+        uses: arduino/setup-protoc@v3
+        with:
+          repo-token: ${{ secrets.GITHUB_TOKEN }}
+
+      - uses: bazelbuild/setup-bazelisk@v3
+
+      - name: Install Clang
+        run: |
+          sudo apt update
+          sudo apt install -y clang
```

Comment:
Same here. Maybe have a look at https://github.com/orgs/eclipse-score/discussions/484 as a rough outline how repositories and CI will be structured in the future.